modules/ROOT/pages/install-self-managed.adoc

```diff
@@ -0,0 +1,248 @@
+| 443 | AMQP over WebSockets | * US control plane: transport-layer.prod.cloudhub.io
+* EU control plane: transport-layer.prod-eu.msap.io | Runtime Fabric message broker for interaction with the control plane.
+| 5044 |TCP (Lumberjack) | * US control plane: dias-ingestor-nginx.prod.cloudhub.io
+* EU control plane: dias-ingestor-nginx.prod-eu.msap.io | Anypoint Monitoring agent for Runtime Fabric.
+| 443 | HTTPS | anypoint.mulesoft.com | Anypoint Platform for pulling assets.
+| 443 | HTTPS | kubernetes-charts.storage.googleapis.com | Kubernetes base charts repository.
+| 443 | HTTPS | docker-images-prod.s3.amazonaws.com | Kubernetes base charts repository.
+| 443 | HTTPS | * US control plane: worker-cloud-helm-prod.s3.amazonaws.com
+* EU control plane: worker-cloud-helm-prod-eu-rt.s3.amazonaws.com worker-cloud-helm-prod-eu-rt.s3.eu-central-1.amazonaws.com | Runtime Fabric version repository. The Runtime Fabric installation uses software from this repository during installation and upgrades.
+| 443 | HTTPS | * US control plane: exchange2-asset-manager-kprod.s3.amazonaws.com
+* EU control plane: exchange2-asset-manager-kprod-eu.s3.amazonaws.com exchange2-asset-manager-kprod-eu.s3.eu-central-1.amazonaws.com |Anypoint Exchange for application assets.
+| 443 | HTTPS | * US control plane: rtf-runtime-registry.kprod.msap.io
+* EU control plane: rtf-runtime-registry.kprod-eu.msap.io | Runtime Fabric Docker repository.
+| 443 | HTTPS | * US control plane: prod-us-east-1-starport-layer-bucket.s3.amazonaws.com prod-us-east-1-starport-layer-bucket.s3.us-east-1.amazonaws.com
+* EU control plane: prod-eu-central-1-starport-layer-bucket.s3.amazonaws.com prod-eu-central-1-starport-layer-bucket.s3.eu-central-1.amazonaws.com | Runtime Fabric Docker image delivery.
+| 443 | HTTPS | * US control plane: runtime-fabric.s3.amazonaws.com
+* EU control plane: runtime-fabric-eu.s3.amazonaws.com | Runtime Fabric Docker repository.
+| 443 | HTTPS | * US control plane: configuration-resolver.prod.cloudhub.io
+* EU control plane: configuration-resolver.prod-eu.msap.io | Anypoint Configuration Resolver.
+|===
+
+=== Certificate Configuration
+
+To allow different endpoints to use mutual TLS authentication to establish a connection, you must configure SSL passthrough to allow the following certificates:
+
+[%header,cols="2*a"]
+.Cerfificate configuration for self-managed kubernetes
+|===
+| Control Plane | Certificates
+| US control plane | transport-layer.prod.cloudhub.io +
+configuration-resolver.prod.cloudhub.io
+| EU control plane | transport-layer.prod-eu.msap.io +
+configuration-resolver.prod-eu.msap.io
+|===
+
+== Step 2: Download the rtfctl Utility
+
+[IMPORTANT]
+====
+The tasks in the section must be performed by an IT administrator.
+====
+
+Runtime Fabric on Self-managed Kubernetes uses the `rtfctl` command-line utility for installation and management tasks. See xref:install-rtfctl.adoc[Install the Runtime Fabric Command Line Tool].
```

Comment:
@sleslie23 there are a few new commands we have added to rtfctl that need to added to this href

Reply:
```
/rtfctl --help
rtfctl controls a Runtime Fabric cluster
	
Find more information at: https://anypoint.mulesoft.com/runtimefabric

Usage:
  rtfctl [command]

Available Commands:
  apply           Apply changes to an RTF configuration
  backup          Backup local Runtime Fabric state
  disk            Print the disk space used by an application
  get             Display one or many resources
  heapdump        Trigger a JVM heap dump
  help            Help about any command
  memory          Display the JVM memory usage in a pod
  package         Create a debugging information package for an application
  ping            Test network connectivity
  report          Generate diagnostics report
  restart         Restart an application
  restore         Restore local Runtime Fabric state from a backup
  status          Display status of the Runtime Fabric
  test            Test Runtime Fabric environment
  threaddump      Print the JVM thread dump for an application
  uninstall       Uninstall Runtime Fabric
  update          Update rtfctl
  upgrade-cluster Upgrade Runtime Fabric cluster
  validate        Validate cluster for Runtime Fabric installation
  version         Print the version information

```

Reply:
Thanks a lot @kmendiratta-crm !!!

Reply:
Fixed.